source/console/minio-console.rst

```diff
@@ -131,316 +131,413 @@ routing rules may require setting a static MinIO Console port. For example,
+
+   .. tab-item:: Trace
+
+      .. image:: /images/minio-console/console-trace.png
+         :width: 600px
+         :alt: MinIO Console Trace
+         :align: center
+
+      The :guilabel:`Trace` section provides HTTP trace functionality for a bucket or buckets on the deployment. 
+      This section provides similar functionality to :mc:`mc admin trace`.
+
+      You can modify the trace to show only specific trace calls.
+      The default is to show only :guilabel:`S3` related HTTP traces.
+      
+      Select :guilabel:`Filters` to open additional filters to apply to trace output, such as restricting the :guilabel:`Path` on which the trace applies to a specific bucket or bucket prefix.
+
+      This tab or its contents may not be visible if the authenticated user does not have the :ref:`required administrative permissions <minio-policy-mc-admin-actions>`.
+
+   .. tab-item:: Watch
+
+      .. image:: /images/minio-console/console-watch.png
+         :width: 600px
+         :alt: MinIO Console Watch
+         :align: center
+
+      The :guilabel:`Watch` section displays S3 events as they occur on the selected bucket. 
+      This section provides similar functionality to :mc:`mc watch`.
 
-The :guilabel:`Heal` section displays the healing status for a bucket. 
-MinIO automatically heals objects and drives when it detects problems, such
-as drive-level corruption or a replacement drive.
+      This tab or its contents may not be visible if the authenticated user does not have the :ref:`required administrative permissions <minio-policy-mc-admin-actions>`.
 
-MinIO does not recommend performing manual healing unless explicitly directed
-by support. 
+   .. tab-item:: Drives
 
-This tab or its contents may
-not be visible if the authenticated user does not have the 
-:ref:`required administrative permissions <minio-policy-mc-admin-actions>`
+      .. image:: /images/minio-console/console-health.png
+         :width: 600px
+         :alt: MinIO Console Healing
+         :align: center
 
-Tools: Diagnostics
-------------------
+      The :guilabel:`Heal` section displays the healing status for a bucket. 
+      MinIO automatically heals objects and drives when it detects problems, such as drive-level corruption or a replacement drive.
 
-.. image:: /images/minio-console/console-diagnostics.png
+      MinIO does not recommend performing manual healing unless explicitly directed
+      by support. 
+
+      This tab or its contents may not be visible if the authenticated user does not have the :ref:`required administrative permissions <minio-policy-mc-admin-actions>`.
+
+Support
+-------
+
+The :guilabel:`Support` section provides an interface for generating health and performance reports.
+You can also register your deployment with MinIO |subnet| to allow upload of health reports directly through the MinIO Console.
```

Comment:
The `|subnet|` replacement includes MinIO already, so we're getting "MinIO MinIO SUBNET" in the rendered file.

Reply:
Fixed